src/test/java/gov/healthit/chpl/aqa/asserts/ViewDeveloperPageAsserts.java

```diff
@@ -146,9 +148,10 @@ public void iDoNotSeeDeveloperMergeLink(final String developerId) {
         boolean developerMergeLinkFound = false;
         try {
             developerMergeLinkFound = ViewDeveloperPage.mergeDeveloperLink(getDriver(), developerId).isDisplayed();
+            fail("Found element when shouldn't have");
         } catch (NoSuchElementException e) {
+            assertTrue(!developerMergeLinkFound, "Developer merge link is available");
```

Comment:
Is there an `assertFalse`? If so, it would be more clear to use that rather than the `!`.

Oh, one other thing. Have you tried adding a "log in" step in this feature temporarily, and seen that the test actually does fail? It's always a good idea to try to make sure your test will actually fail sometimes, otherwise you're can't be sure you're exercising the system correctly

Reply:
Yes, I tried adding a "log in" step and verified that the test fails. Here is the error that I see:- **java.lang.AssertionError: Found element when shouldn't have**